apps/backend/.env.example

```diff
@@ -4,6 +4,7 @@
 NODE_ENV=development
 PORT=3001
 HOST=localhost
+JWT_SECRET_KEY=someSecretKey
```

Comment:
Why not JWT_SECRET? Adding _KEY at the end feels redundant

Reply:
It seems to be fine, just a question of tastes